contexts/provider-context.js

```diff
@@ -15,63 +15,66 @@ export function ProviderProvider({ children }) {
   const nodeIsHealthy = async (url) => {
     var myHeaders = new Headers();
     myHeaders.append("Content-Type", "application/json");
-  
-    var raw = JSON.stringify({"jsonrpc":"2.0","id":1,"method":"health.getLiveness"});
-  
+
+    var raw = JSON.stringify({ "jsonrpc": "2.0", "id": 1, "method": "health.getLiveness" });
```

Comment:
const raw = JSON.stringify({ "jsonrpc": "2.0", "id": 1, "method": "health.getLiveness" });

Reply:
honestly we need to review all our code for this kind of thing, this will not change the result of the function for now, so I will discard this and we make a cleanup task later